src/components/sidebar/tabs/ModelLibrarySidebarTab.vue

```diff
@@ -137,6 +137,31 @@ const renderedRoot = computed<TreeExplorerNode<ComfyModelDef>>(() => {
           return 'pi pi-file'
         }
       },
+      getBadgeText: (node: TreeExplorerNode<ComfyModelDef>) => {
+        if (node.leaf) {
+          return ''
+        }
+        if (node.children?.length === 1) {
+          const onlyChild = node.children[0]
+          if (onlyChild.data?.is_fake_object) {
+            if (onlyChild.data.name === '(No Content)') {
+              return '0'
+            } else if (onlyChild.data.name === 'Loading') {
+              return '?'
+            }
+          }
+        }
+        const getTotalLeaves = (node: TreeExplorerNode<ComfyModelDef>) => {
```

Comment:
Let's lift the totalLeaves calculation to `buildTree` so that we don't need to traverse the subtree again.